dashboard/python_scripts/compute_stats.py

```diff
@@ -0,0 +1,166 @@
+import os
+import json
+import logging
+import subprocess
+from argparse import ArgumentParser
+from pathlib import Path
+from statistics import mean
+
+import datasets
+from datasets import config, load_from_disk
+from datasets.utils.logging import set_verbosity_info
+
+set_verbosity_info()
+logger = logging.getLogger(__name__)
+
+
+def get_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--dataset-path",
+        type=str,
+        required=True,
+        help="path to the parquet dataset folder",
+    )
+    parser.add_argument(
+        "--save-path-stats-json", type=str, help="Where to save the stats json."
+    )
+    parser.add_argument(
+        "--save-path-stats-full-json", type=str, help="Where to save the stats json."
+    )
+    parser.add_argument(
+        "--save-batch-size", type=int, required=True, help="Batch size when writing."
+    )
+    parser.add_argument("--use-datasets-caching", action="store_true")
```

Comment:
We never us it. Let's remove it.

Reply:
In fact, it seems to me that I used  `use_datasets_caching`